doc/research/paf24/general/leaderboard_summary.md

```
@@ -0,0 +1,119 @@
+# Leaderboard summary
+
+**Summary:** This document depicts general informations regarding the leaderboard as a quick overview in a more condensed form.
+
+ - [General Information](#general-information)
```

Markdownlint

Expected: 0; Actual: 1
Unordered list indentation

(MD007, ul-indent)